autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleScalingActivityV1Controller.java

```diff
@@ -0,0 +1,117 @@
+package com.sequenceiq.periscope.controller;
+
+import static java.time.temporal.ChronoUnit.MINUTES;
+
+import java.time.Instant;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Controller;
+
+import com.sequenceiq.authorization.annotation.CheckPermissionByAccount;
+import com.sequenceiq.authorization.annotation.CheckPermissionByResourceCrn;
+import com.sequenceiq.authorization.annotation.CheckPermissionByResourceName;
+import com.sequenceiq.authorization.annotation.ResourceCrn;
+import com.sequenceiq.authorization.annotation.ResourceName;
+import com.sequenceiq.authorization.resource.AuthorizationResourceAction;
+import com.sequenceiq.cloudbreak.auth.security.internal.TenantAwareParam;
+import com.sequenceiq.periscope.api.endpoint.v1.DistroXAutoScaleScalingActivityV1Endpoint;
+import com.sequenceiq.periscope.api.model.DistroXAutoscaleScalingActivityResponse;
+import com.sequenceiq.periscope.converter.DistroXAutoscaleScalingActivityResponseConverter;
+import com.sequenceiq.periscope.model.NameOrCrn;
+import com.sequenceiq.periscope.service.AutoscaleRestRequestThreadLocalService;
+import com.sequenceiq.periscope.service.ScalingActivityService;
+
+@Controller
+public class DistroXAutoScaleScalingActivityV1Controller implements DistroXAutoScaleScalingActivityV1Endpoint {
+
+    public static final Integer MINUTES_TO_SUBTRACT = 60;
+
+    @Inject
+    private DistroXAutoscaleScalingActivityResponseConverter distroXAutoscaleScalingActivityResponseConverter;
+
+    @Inject
+    private ScalingActivityService scalingActivityService;
+
+    @Inject
+    private AutoscaleRestRequestThreadLocalService threadLocalService;
+
+    @Override
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.DESCRIBE_DATAHUB)
+    public DistroXAutoscaleScalingActivityResponse getScalingActivityUsingActivityCrn(@TenantAwareParam String activityCrn) {
```

Comment:
what is activityCrn?

Reply:
Activity Crn is the unique Crn value for the scaling activity.